lib/serverrequestcontext.go

```diff
@@ -202,10 +204,17 @@ func (ctx *serverRequestContextImpl) verifyX509Token(ca *CA, authHdr, method, ur
 	if !checked {
 		return "", caerrors.NewHTTPErr(401, caerrors.ErrCertRevokeCheckFailure, "Failed while checking for revocation")
 	}
+
 	if expired {
-		return "", caerrors.NewAuthenticationErr(caerrors.ErrCertExpired,
-			"The certificate in the authorization header is a revoked or expired certificate")
+		log.Debugf("Expired Certificate")
+		if ctx.endpoint.Path == "reenroll" && ctx.ca.Config.CA.IgnoreCertExpiry {
```

Comment:
...and if we create the well-named variable above, we can re-use it here instead of re-doing the logic.